api-schema/src/main/java/org/stellar/anchor/api/rpc/RpcRequest.java

```diff
@@ -0,0 +1,13 @@
+package org.stellar.anchor.api.rpc;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class RpcRequest {
+  private Object id;
+  private String jsonrpc;
+  private String method;
+  private Object params;
```

Comment:
I'm guessing it's not possible to make it generic, on what step this going to be converted to an actual object?